.gitattributes

```diff
@@ -12,3 +12,26 @@
 *.jpg binary
 
 *.map linguist-generated
+
+.editorconfig export-ignore
+.eslintignore export-ignore
+.eslintrc export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.stylelintignore export-ignore
+.stylelintrc.json export-ignore
+.wp-env.json export-ignore
+composer.json export-ignore
+composer.lock export-ignore
+CONTRIBUTING.md export-ignore
+CONTRIBUTORS.md export-ignore
+package-lock.json export-ignore
+package.json export-ignore
+phpcs.xml.dist export-ignore
+phpstan.neon export-ignore
+postcss.config.js export-ignore
+README.md export-ignore
+style.css.map export-ignore
+.github/ export-ignore
+assets/css/*.map export-ignore
+assets/sass/ export-ignore
```

Comment:
Please also exclude the `readme.txt` file.

Reply:
Don't know `readme.txt` is required or not others LGTM:+1:

Reply:
readme.txt is required for themes hosted in the w.org repository, that's why it wasn't added.

Thank you for the review!